lessons-from-hiring-manager-interviews.rst

```diff
@@ -0,0 +1,165 @@
+
+.. _Joel Spolsky had it right: https://www.inc.com/magazine/20070501/column-guest.html
+
+
+Recruiters can be worth it
+--------------------------
+
+In each interview, we asked whether the company used recruiters or recruiting related services (eg. Hired, Triplebyte).
+Universally, hiring managers brought up how expensive recruiters are,
+but most of the people we talked to who had actually used recruiters in the past had a positive impression overall.
+Hiring takes a ton of time and typically takes it from senior folks at the company.
+At smaller startups, that's frequently the CEO or CTO.
+Even though the price is high, recruiters who help speed that process along are worth it.
+
+I think the quote that really summed it up was from a VP of engineering at a startup that just raised a Series B.
+He's trying to grow his team by 10 developers this year (about a 50% increase).
+His experience hiring his first couple developers through a 2-person boutique recruiting firm was great and
+he would use them for hiring everyone "if hiring 10 people didn't mean $100k+" in fees alone.
+
+Full disclosure: Triplebyte is an advertiser on Read the Docs.
+
+
+Filtering resumes isn't a real problem
+--------------------------------------
+
+Every single hiring manager we talked to complained about the time they spent filtering resumes from applicants.
+Some choice quotes I heard included "95% of applicants weren't a fit" even after only reviewing the resume
+and a hiring manager who posted to a very popular remote-only job board told me he got "literally one thousand applications".
+While I believe this is a legitimate complaint,
+I think `Aline Lerner said it best`_ when she said "Engineering hiring isn't a filtering problem. It's a sourcing problem."
+Filtering is just a time-consuming task that hiring managers directly deal with.
+
+The real problem here is that **the number of applicants to a job post is a vanity metric**.
+What hiring managers are looking for is qualified applicants, which is harder to gauge without some additional work.
+One of the problems here is that a lot of jobs sites make it really easy for candidates to apply to lots of jobs.
+This is understandable from their perspective as more applicants makes it look like their platform is more valuable.
+I'm not trying to talk trash on these platforms as there's no reason for them to erect artificial barriers to the application funnel,
+but hiring managers told us they want candidates who want to work for their company
+not somebody who is applying to every single job out there. More on that in a future post.
+
+Instead of building product here, I think hiring managers just need a few tweaks
+to get more of a signal of candidate quality and intent in their applications.
+A few of the hiring managers we talked to described how they did this.
+Multiple hiring managers mentioned how they required a cover letter that answered specific questions ("why us" was the most popular).
```

Comment:
Might be good to put these in a list to make it easier to skim. Otherwise it will likely get caught on a skim.